ledger/ledger-api-common/src/main/scala/com/digitalasset/platform/server/api/validation/FieldValidations.scala

```diff
@@ -10,8 +10,8 @@ import com.daml.ledger.api.domain
 import com.daml.ledger.api.domain.LedgerId
 import com.daml.ledger.api.v1.value.Identifier
 import com.daml.ledger.api.validation.ValidationErrors._
-import com.daml.ledger.participant.state.index.ResourceAnnotationValidation
-import com.daml.ledger.participant.state.index.ResourceAnnotationValidation.{
+import com.daml.ledger.participant.localstore.utils.ResourceAnnotationValidation
```

Comment:
This is an inverted dependency. `ledger-api-common` is a utility library with minimum dependencies that in turn is used in many places such as `daml-script` and `daml-lf`. I would suggest to move the `ResourceAnnotationValidation` into here.

Reply:
Ok, moving `ResourceAnnotationValidation` to `ledger-api-common`